src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -34,7 +34,15 @@ class DaosCoreTest(TestWithServers):
     Runs the daos_test subtests with multiple servers.
     :avocado: recursive
     """
-    subtest_name = None
+    def __init__(self, *args, **kwargs):
+        super(DaosCoreTest, self).__init__(*args, **kwargs)
+
+        self.subtest_name = None
+
+        self.test_timeout = self.params.get("test_timeout",
```

Comment:
Is this value used outside the scope of the `__init__()` method? If not it could just be `test_timeout` (w/o self)

Reply:
Lines 44-45 below.

Reply:
Yes, but that is still within the scope of `__init__()`.  If we don't need to reference `self.test_timeout` in `setUp()`, `tearDown()`, or any other class method, then it should be defined as `test_timeout`:

```
test_timeout = self.params.get("test_timeout", '/run/daos_tests/Tests/*')
if test_timeout:
    self.timeout = test_timeout
```

Reply:
Ah yes, you are correct.